graph/client/src/app/feature-projects/machines/project-graph.machine.ts

```diff
@@ -254,10 +254,10 @@ export const projectGraphMachine = createMachine<
         //   name: 'route',
         // });
 
-        if (event.type === 'setProjects') {
-          ctx.workspaceLayout = event.workspaceLayout;
-          ctx.affectedProjects = event.affectedProjects;
-        }
+        // if (event.type === 'setProjects') {
+        ctx.workspaceLayout = event.workspaceLayout;
+        ctx.affectedProjects = event.affectedProjects;
+        // }
```

Comment:
@philipjfulcher Is it safe to completely remove this `if` statement? 

I changed the `updateGraph` event to also have these properties. Is there a reason they didn't have them that I'm unaware of?

Reply:
Yes, this is safe. The reason we checked before is due to some typing issues with xstate. This action can handle two different events, but xstate can't figure out the event that trigger this action. 

```typescript
        if (event.type !== 'setProjects' && event.type !== 'updateGraph')
          return;
```

This block at the top of the action narrows down the types of events it needs. Since the interface for the events are different previously, this if statement handled the different. Now that you've updated them to be similar interfaces, the check is no longer necessary. Go ahead and remove the commented code.